blocks/library/gallery/editor.scss

```diff
@@ -13,6 +12,26 @@
 	&.is-transient img {
 		@include loading_fade;
 	}
+
+	.blocks-rich-text__inline-toolbar {
+		margin-top: -1.8em;
+	}
+
+	.blocks-rich-text {
+		position: absolute;
+		width: 100%;
+	}
+
+	.blocks-rich-text figcaption:last-of-type {
+		position: relative;
```

Comment:
What does this do?

Reply:
I see the reason now, too bad about the rich text wrapper. Still note understanding the `last-of-type` though.

Reply:
We want position relative, to figcaption elements in the editor except when we have a placeholder. In this cases, RichText creates 2 figcaption the first should use absolute and second relative like in the other cases. In order to not overwrite the property of the first figcaption when placeholder is visible, this rule was used. Relying on attributes like data-is-placeholder-visible caused blinks because the attributes are not immediately added to the dom. So I updated the rule and in this version, it seems there are no blinks.

Reply:
Could you leave a comment?

Reply:
Comment added.